app.py

```diff
@@ -78,12 +79,20 @@ def spool_info():
     last_ams_config = getLastAMSConfig()
     ams_data = last_ams_config.get("ams", [])
     vt_tray_data = last_ams_config.get("vt_tray", {})
+    spool_list = fetchSpools()
+    
+    issue = False
+    #TODO: Fix issue when external spool info is reset via bambulab interface
+    augmentTrayDataWithSpoolMan(spool_list, vt_tray_data, trayUid(EXTERNAL_SPOOL_AMS_ID, 254))

```

Comment:
Please use constant instead of hard coded 254

Reply:
I will do a test print to view the original messages from the printer